.gitmodules

```diff
@@ -1,8 +1,8 @@
 [submodule "aion_fastvm"]
     path = aion_fastvm
     url = https://github.com/aionnetwork/aion_fastvm
-    branch = master
+    branch = gradle  
 [submodule "aion_api"]
     path = aion_api
     url = https://github.com/aionnetwork/aion_api
-    branch = master
+    branch = gradle 
```

Comment:
we wait for the PR in vm and api repo merged, then change branch to gradle

Reply:
yeah, sounds good